changes default hardware handler to hp_sw from hp-sw. This change has already been done upstream in 2009
Index: multipath-tools/libmultipath/hwtable.c
===================================================================
--- multipath-tools.orig/libmultipath/hwtable.c	2011-10-30 12:37:27.902672064 +0530
+++ multipath-tools/libmultipath/hwtable.c	2011-10-30 12:40:22.942007814 +0530
@@ -85,7 +85,7 @@
 		.product       = "HSG80",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "1 queue_if_no_path",
-		.hwhandler     = "1 hp-sw",
+		.hwhandler     = "1 hp_sw",
 		.selector      = DEFAULT_SELECTOR,
 		.pgpolicy      = GROUP_BY_PRIO,
 		.pgfailback    = FAILBACK_UNDEF,
@@ -116,7 +116,7 @@
 		.product       = "(MSA|HSV)1.0.*",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "1 queue_if_no_path",
-		.hwhandler     = "1 hp-sw",
+		.hwhandler     = "1 hp_sw",
 		.selector      = DEFAULT_SELECTOR,
 		.pgpolicy      = GROUP_BY_PRIO,
 		.pgfailback    = FAILBACK_UNDEF,
